client/state/themes/actions/activate-theme.js

```diff
@@ -35,6 +36,7 @@ export function activateTheme(
 			.post( `/sites/${ siteId }/themes/mine`, {
 				theme: themeId,
 				...( dontChangeHomepage && { dont_change_homepage: true } ),
+				...( isEnabled( 'themes/theme-switch-persist-template' ) && { persist_template: true } ),
```

Comment:
Super nit: since we have `dont_change_homepage`, can we also make this param more explicit, e.g. `persist_homepage_template`?

Reply:
Good idea, I also updated the diff for this change as well